app/components/Icons/EmojiIcon.tsx

```diff
@@ -17,18 +17,28 @@ type Props = {
 export default function EmojiIcon({ size = 24, emoji, ...rest }: Props) {
   return (
     <Span $size={size} {...rest}>
-      {emoji}
+      <SVG size={size} emoji={emoji} />
     </Span>
   );
 }
 
 const Span = styled.span<{ $size: number }>`
   font-family: ${s("fontFamilyEmoji")};
-  display: inline-flex;
-  align-items: center;
-  justify-content: center;
-  flex-shrink: 0;
+  display: inline-block;
   width: ${(props) => props.$size}px;
   height: ${(props) => props.$size}px;
-  font-size: ${(props) => props.$size - 10}px;
 `;
+
+const SVG = ({ size, emoji }: { size: number; emoji: string }) => (
+  <svg width={size} height={size} xmlns="http://www.w3.org/2000/svg">
```

Comment:
Rendering the emoji inside an `svg` for a consistent alignment across retina & non-retina displays.